build.proj

```diff
@@ -78,28 +78,6 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
-  <!-- TODO: Can we move this archive test build to BuildTools -->
-  <UsingTask TaskName="ZipFileCreateFromDependencyLists" Condition="'$(ArchiveTests)' == 'true'" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
```

Comment:
I think this target needs some more work, especially since we need to change the way we zip up the dependencies for Helix. This doesn't need to be in this changeset.